src/elements/List/List.js

```diff
@@ -1,32 +1,44 @@
-import React, { Component, PropTypes } from 'react'
-import classNames from 'classnames'
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
 import META from '../../utils/Meta'
+import {getUnhandledProps} from '../../utils/propUtils'
 import ListItem from './ListItem'
 
-export default class List extends Component {
-  static propTypes = {
-    children: PropTypes.node,
-    className: PropTypes.string,
-  }
-
-  static _meta = {
-    library: META.library.semanticUI,
-    name: 'List',
-    type: META.type.element,
-  }
-
-  static Item = ListItem
-
-  render() {
-    const classes = classNames(
-      'ui',
-      this.props.className,
-      'list'
-    )
-    return (
-      <div {...this.props} className={classes}>
-        {this.props.children}
-      </div>
-    )
-  }
+function _List(props) {
```

Comment:
Since this is your first PR, I'll do a little extra explaining 😉 This is an inline comment, you can leave these by clicking a line number in the diff (use the File tab).

When a comment is left on a specific line like this, it is because we want to discuss the code on this line or just surrounding it.  Here goes.

Underscore names are reserved for private components and variables.  Let's rename this to `List` instead.

```diff
-function _List(props) {
+function List(props) {
```